other/field-types/slug.md

```diff
@@ -0,0 +1,13 @@
+# `slug`
+
+`slug` adds a slug field to the schema for specifying the contextual location of content in your site. Usually there is only one, named `slug`, and it is already part of your schema when extending pieces or custom pages.
+
+By default slugs are sanitized by the [sluggo](https://github.com/punkave/sluggo) module. This can be changed by overriding the `apos.utils.slugify` method.
```

Comment:
/apostrophecms/sluggo

Reply:
People do ask this question but we could remove it if you want to go for brevity, if they really want to know that badly there's the source

Reply:
Yeah, seems unnecessary for this level of the documentation.